neuraxle/pipeline.py

```diff
@@ -127,16 +127,29 @@ def fit_data_container(self, data_container):
 
         return new_self
 
-    def inverse_transform(self, processed_outputs) -> Any:
+    def inverse_transform(self, processed_outputs, expected_outputs=None) -> Any:
```

Comment:
really? add expected?

Reply:
I don't think we need to add the `expected_outputs`. Please remove this except if you have a very good reason.